ref/general/authentication.adoc

```diff
@@ -17,54 +17,57 @@ In order for an application to be secure, it must be able to determine who its u
 
 Open Liberty authenticates users by collecting credentials from them, such as a username and password, and checking these credentials against a configured registry of users. Authentication can be carried out a number of different ways, depending on the details of an application and the resources that are available to it. For microservice-based applications, authentication must be lightweight and configurable, since any user request might need to authenticate to multiple services. In many cases, a user's credentials are placed in a token that can be passed from the server to different microservices to authenticate requests for protected resources.
 
-For example, a user of a music streaming service might want to listen to a song, learn about the artist, and get recommendations for similar artists. Each of these tasks would require access to a different service, but having to authenticate for each of them separately uses unnecessary time and resources. Open Liberty supports a number of different strategies for single sign-on (SSO) authentication so that a user or service must authenticate only once to access the various resources in an application. For example, the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] feature enables users of an application that is running on Open Liberty to authenticate through their social media login. After authentication, an application can either obtain a JSON Web Token (JWT) from the social media service or associated OpenID Connect provider, or create a new one. This JWT can then be used to authenticate the user to any service in the application.
+For example, a user of a streaming music service might want to listen to a song, learn about the artist, and get recommendations for similar artists. Each of these tasks would require access to a different service, but authenticating to each service separately uses unnecessary time and resources. Open Liberty supports a number of different strategies for single sign-on (SSO) authentication so that a user or service authenticates only one time to access the various resources in an application. For example, the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login] feature enables users of an application that is running on Open Liberty to authenticate through their social media login. After authentication, an application can either obtain a JSON Web Token (JWT) from the social media service or associated OpenID Connect provider, or create a new one. This JWT can then authenticate the user to any service in the application.
```

Comment:
In 'authenticating to each service separately uses unnecessary time and resources',
Instead of 'unnecessary' can you say 'more'?